### PR TITLE
Feature/iot_1239_simulator

### PR DESCRIPTION
**Pull request by augusthjerrild**
Simulator for gateway and device

### CODE REVIEW

File: Simulator/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM ubuntu
```
Comment (GufCab) [inline Simulator/Dockerfile:1]: Folder name should be lowercased to "simulator" to match with other folders.

Comment (GufCab) [inline Simulator/Dockerfile:1]: Can "Dockerfile" also be lowercased to dockerfile or is this the standard?

**Review by GufCab — APPROVED** [review]
Fix the remaining comments, then it's ready for merge.

File: Simulator/Dockerfile (hunk 1 above)
Comment (augusthjerrild) [reply Simulator/Dockerfile:1]: Uppercase d is the standard in this project

**Pull request merged**